src/main/java/org/folio/bulkops/domain/converter/EffectiveCallNumberComponentsConverter.java

```diff
@@ -23,9 +28,11 @@ protected String convertToWrite(Object value) {
       return EMPTY;
     }
     var components = (EffectiveCallNumberComponents) value;
-    return String.join(ARRAY_DELIMITER, isEmpty(components.getCallNumber()) ? EMPTY : escape(components.getCallNumber()),
-      isEmpty(components.getPrefix()) ? EMPTY : escape(components.getPrefix()),
-      isEmpty(components.getSuffix()) ? EMPTY : escape(components.getSuffix()), escape(ItemReferenceService.service()
-        .getCallNumberTypeNameById(components.getTypeId())));
+    List<String> comps = new ArrayList<>();
```

Comment:
plz look at how effective call number is generated in inventory https://github.com/folio-org/stripes-util/blob/master/lib/effectiveCallNumber.js#L27